docs/resources/appdata_dsource.md

```diff
@@ -0,0 +1,176 @@
+# Resource: <resource name> delphix_appdata_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Virtualization Engine uses to create and update virtual copies of your database. 
+A dSource is created and managed by the Delphix Continuous Data Engine.
+
+
+The dSource resource allows terraform to CREATE, READ and DELETE dSources. 
+Update operation is not supported. The supported parameters are listed below.
+
+## Example Usage
+dSource Linking can be done in 3 methods , the parametes for these methods wary based on choice.
```

Comment:
This doesn't make sense to me. Also, some spelling spacing, and capitalization errors.

Reply:
Do you think we should rephrase this? basically, core attributes remain the same but in the `parameters` section we can provide either of the 3 linking mechanisms , as shown in the example

Reply:
The linking of a dSource can be configured through various ingestion approaches. Each configuration is customized to the connector and its supported options. The three PostgreSQL examples below show working examples.